src/components/Toolbar/index.js

```diff
@@ -126,6 +127,24 @@ export default class Toolbar extends Component {
     );
   }
 
+  renderInfo() {
+    return (
+      <div className={'toolbar--info'}>
+        <Popup icon="fa-info-circle" onBlur={this.handleBlur}>
+          <strong>How to Enter Answers:</strong>
+          <ul>
+            <li>
+              You can click a square once to enter an answer, and click that same square again to switch
```

Comment:
Maybe remove `You can `? I'm personally more used to instructions being imperative, e.g. `Click a square once to enter an answer, and click that same square again to switch`